CoreVisualSchema/src/au/gov/asd/tac/constellation/graph/schema/visual/attribute/objects/Hashmod.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.graph.schema.visual.attribute.objects;
+
+import au.gov.asd.tac.constellation.utilities.color.ConstellationColor;
+
+/**
+ * A Hashmod that will allow setting of attributes from a key attribute's value
+ *
+ * @author CrucisGamma
+ */
+public class Hashmod {
```

Comment:
Is there a reason why this couldn't be put in with the other Hashmod.java?

Reply:
This should have been deleted.  I was basing it off of banner which requires an actual element.  I took it out in my second pull request (I thought).